### ISSUE TITLE
"Can't access necessary data" notice flicker when activating Analytics via GTM setup

### ISSUE THREAD

**Issue by aaemnnosttv** [google/site-kit-wp#5244]
## Bug Description

When setting up Analytics via GTM setup, the unsatisfied scopes notice briefly flickers before the user is taken to the OAuth consent screen.

https://user-images.githubusercontent.com/1621608/169626401-24e4b88e-3bc6-46b4-99f8-b05a3554fbb2.mp4

## Steps to reproduce

1. Connect GTM using a container that has an Analytics property in the container
2. Choose the primary option to "Continue to Analytics setup"
3. Notice flicker on submit

## Screenshots

![image](https://user-images.githubusercontent.com/1621608/169626410-f5e183c6-2d26-4707-9e3a-3ca8e5321e5e.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by kuasha420** [comment]
@wpdarren Yep, feel free to create a new issue for this one, the reason for this flicker doesn't appear to be similar. Nice spot though!

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- I can see that flicker no longer appears when you click on the 'continue to Analytics setup' button. I was able to recreate this on the latest release, but on develop the flicker is fixed.   Found an issue with the Analytics setup as part of the GTM process but will create this as an issue since it's out of scope for this ticket. 

https://user-images.githubusercontent.com/73545194/175474981-aeb17077-5765-43cc-bcab-286f76714882.mp4
